.travis.yml

```diff
@@ -1,6 +1,5 @@
-language: generic
-
-sudo: false
+sudo: true
```

Comment:
Why did you do this? Sudoless container-based infrastructure on Travis is faster.

Reply:
According to research of our DevOps team, `sudo: true` makes builds faster. If this it not the case, you should contact our DevOps team, I'm just using instructions from the article. Read Travis CI in Haskell article by our team.